src/pages/Admin/Charity/Banking/ExpectedFunds.tsx

```diff
@@ -8,9 +11,38 @@ type Props = {
 export default function ExpectedFunds({ disabled, onChange }: Props) {
   return (
     <div className="field">
-      <Label htmlFor="amount" required aria-required>
-        What is the amount of donations you expect to receive on our platform?
-      </Label>
+      <div className="flex sm:gap-2 items-center mb-1">
+        <Label htmlFor="amount" required aria-required>
+          What is the amount of donations (in USD) you expect to receive monthly
+          on our platform?
+        </Label>
+        <Popover className="relative">
```

Comment:
Added popover as per the new requirement, see https://linear.app/angel-protocol/issue/AP-859/set-an-actual-calculation-to-get-the-estimate-payout-amount-when#comment-7b49d941